src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/SwaggerIgnoreAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+// ReSharper disable once CheckNamespace
+namespace Swashbuckle.AspNetCore.Annotations
+{
+    /// <summary>
+    /// Causes the annotated member to be ignored during schema generation.
+    /// Does not alter serialization behavior.
+    /// </summary>
+    /// <remarks>
+    /// Can be used in combination with <see cref="System.Text.Json.Serialization.JsonExtensionDataAttribute"/>
+    /// to capture and invalidate unsupported properties.
+    /// </remarks>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Parameter | AttributeTargets.Property)]
```

Comment:
System.Text.Json also supports fields for serialisation, so should this too?

Reply:
I actually didn't realize JsonSerializerOptions had been added for this in .NET 5, however it looks as though significant portions of the Swashbuckle codebase only consider properties (e.g. looking at the CustomAttributes extension used elsewhere), so I'm hesitant to take on this change.